config/packages/nelmio_api_doc.yaml

```diff
@@ -1,22 +1,9 @@
-#nelmio_api_doc:
-#    exclude_sections:     []
-#    default_sections_opened:  true
-#    motd:
-#        template:             'NelmioApiDocBundle::Components/motd.html.twig'
-#    request_listener:
-#        enabled:              true
-#        parameter:            _doc
-#    swagger:
-#        api_base_path:        /api
-#        swagger_version:      '1.2'
-#        api_version:          '0.1'
-#        info:
-#            title:                Symfony2
-#            description:          'My awesome Symfony2 app!'
-#            TermsOfServiceUrl:    null
-#            contact:              null
-#            license:              null
-#            licenseUrl:           null
-#    cache:
-#        enabled:              false
-#        file:                 '%kernel.cache_dir%/api-doc.cache'
+nelmio_api_doc:
+    documentation:
+        info:
+            title: Pelagos
+            description: This is an awesome app!
```

Comment:
Not sure what the description should be, or the version?

Reply:
we can start with 1.0.0 for version and its anyways for internal viewing

Reply:
The API is public, some people may make use of it. However, we don't publish it the URL publicly I thnk.